docs/visualization/lupton_rgb.rst

```diff
@@ -67,4 +70,19 @@ Lupton et al. (2004):
    # this scaling is very similar to the one used in Lupton et al. (2004)
    rgb = make_lupton_rgb(i_new, r_new, g.data, Q=10, stretch=0.5, filename="ngc6976.jpeg")
 
-TODO: show example files in this document right here? How?
+This will produce the following two images.
+
+.. figure:: https://astropy.stsci.edu/data/visualization/ngc6976-default.jpeg
```

Comment:
This (and the example below) should `http://data.astropy.org/visualization/ngc6976-default.jpeg` - astropy.stsci.edu/data is not the "canonical" url, even though it's what data.astropy.org go to